### ISSUE TITLE
Smaller sound support

### ISSUE THREAD

**Issue by Marootc** [SWivid/F5-TTS#1078]
### Checks

- [x] This template is only for research question, not usage problems, feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I am using English to submit this issue to facilitate community communication.

### Question details

Hello, I have some problems with fine-tuning, so I would like to ask you for advice. I fine-tuned the whisper data, which includes English and Chinese whispers. The data set is about 60 hours. When I was doing step 200k, I tried inference. Most of the fine-tuning content can be effectively inferred, but many of the inferences with small sounds have no audio.
It cannot generate very low sound.
![Image](https://github.com/user-attachments/assets/02413fa8-82bf-4055-b850-e6292de79ed4)

![Image](https://github.com/user-attachments/assets/1289b925-fa05-4000-8a72-544914199456)
It is effective for the following
![Image](https://github.com/user-attachments/assets/21e1dabd-8cf2-4496-be13-97f56bed4a7b)
Are there any other settings I need to modify when fine-tuning?
Although I haven't completed the complete training, I think it should be able to understand this whisper.
Epoch 18/200:  58%|█████▊    | 6430/11114 [26:56<19:18,  4.04update/s, loss=0.355, update=195367]

**Comment by SWivid** [comment]
Hi @Marootc , what is your ema and remove_silence setting here? Maybe turn off use_ema and remove_silence would be better.
If still not able to get some good results, I think train from scratch with 60 hours whisper data is worth trying (it could be hard to tune the model out of the pretrained data distribution where whispers are not very learned before).

**Comment by Marootc** [comment]
@SWivid Thanks for your reply, yes I tried turning off ema and remove_silence and it didn't seem to change the results, except for very small sound inferences. I will try training from scratch.

**Comment by Marootc** [comment]
@SWivid Hello, what is the option in finetune_gradio.py for training from scratch?

**Comment by SWivid** [comment]
No idea, we mostly use `train.py`
